.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 1
```

Comment:
LGTM!   This is awesome!  I really wanted this at my last job, but we didn't have the gitlab license that supported it!  So excited to have this turned on!

Reply:
Let's see if it'll just work 😃